pkgs/unified_analytics/lib/src/utils.dart

```diff
@@ -105,6 +106,20 @@ Directory? getHomeDirectory(FileSystem fs) {
   return fs.directory(home);
 }
 
+/// Function used to convert a string into an integer value
+int intFromString(String input) {
```

Comment:
What's the meaning of this value? Are you just trying to hash the input string? If so, can we just use `input.hashCode`?

Reply:
I was unsure about the probability distribution using the hashcode for a given string.  But seeing that it is a hash that is getting generated from the code units, it be okay to swap them.

Doing some testing with the example file still shows the sample rate is being observed